tests/net/context/src/main.c

```diff
@@ -978,10 +817,10 @@ static bool net_ctx_recv_v6_timeout(void)
 
 	expecting_cb_failure = false;
 
-	return !cb_failure;
+	zassert_true((!cb_failure), NULL);
```

Comment:
you can write this as zassert_true(!cb_failure, ...), the extra () are not needed

Reply:
Done